frontend/src/components/eventselect/reducer/analyticsEventSelectReducer.ts

```diff
@@ -0,0 +1,208 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { OptionDefinition } from '@cloudscape-design/components/internal/components/option/interfaces';
```

Comment:
update the file name to first capital character

Reply:
Because this is a `.ts` file, should we change the file to first capital character. or More this file the another folder?